webapp/src/components/github_assignee_selector/github_assignee_selector.jsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+import IssueAttributeSelector from 'components/issue_attribute_selector';
+
+export default class GithubAssigneeSelector extends PureComponent {
+    static propTypes = {
+        repo: PropTypes.string.isRequired,
```

Comment:
Since `state.repo` in the parent is an object, can we change this prop name to `repoName` to avoid confusion? Same with the other similar child components.

Reply:
@mickmister Done: https://github.com/mattermost/mattermost-plugin-github/pull/343/commits/e274b08cb89d1c958f9fd2f3ea252d2a19ab26f5